.golangci.yml

```diff
@@ -64,14 +64,6 @@ linters-settings:
           - (github.com/golangci/golangci-lint/pkg/logutils.Log).Warnf
           - (github.com/golangci/golangci-lint/pkg/logutils.Log).Errorf
           - (github.com/golangci/golangci-lint/pkg/logutils.Log).Fatalf
-
-    # enable or disable analyzers by name
```

Comment:
Do these conflicts with any of the other changes?

Reply:
Not sure I understand.

Reply:
why these are being removed? are the same covered by `govet`?

Reply:
Actually the logic was to enable only `atomicalign` in `govet` if I understand it correctly. What I did was to keep all default enabled checks in `govet`

Reply:
I read the source code, I was wrong Enable behavior is "enable + default". Fixed in https://github.com/open-telemetry/opentelemetry-collector/pull/739 by enabling all analyzers.